nixos/lib/utils.nix

```diff
@@ -21,6 +21,22 @@ rec {
       config.networking.hostName
     ]);
 
+  # Returns first service from /etc/nixos/services.json
+  # that matches the given name or null, if nothing matches.
+  # XXX: check for multiple matches here?
```

Comment:
Suggestion for the XXX: stop evaluation and break (causing no further config changes) which will cause no direct service interruption but will alert support personnel at an appropriate time.